src/EA.Iws.Web/Areas/AdminImportAssessment/Views/DeleteTransaction/Confirm.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@model EA.Iws.Web.Areas.AdminImportAssessment.ViewModels.DeleteTransaction.ConfirmViewModel
+@using Resource = EA.Iws.Web.Areas.AdminImportAssessment.Views.DeleteTransaction.ConfirmResources
+@using EnumHelper = EA.Prsd.Core.Helpers.EnumHelper
+
+@{
+    ViewBag.Title = Resource.MainHeading;
+}
+
+<h1 class="heading-large">@Resource.MainHeading</h1>
+
+<div class="form-group">
+    <p>@Resource.subTitle</p>
+    <div class="gird-row">
+        <div class="column-one-third bold-small">
+            @Resource.TransactionType </br>
+            @Resource.Date  </br>
+            @Resource.Amount  </br>
+            @Resource.Type </br>
+            @Resource.Receipt  </br>
+            @Resource.Comments  </br>
+        </div>
+        <div class="column-two-thirds">
+            @Model.Transaction </br>
+            @Html.Gds().DisplayShortDateFor(m => Model.Date) </br>
+            @Model.Amount.ToString("C") </br>
+            @Model.Type </br>
+            @Model.ReceiptNumber </br>
+            @(Model.Comments == string.Empty || Model.Comments == null ? "None" : Model.Comments) </br>
+        </div>
```

Comment:
This layout could probably be better achieved using a table rather than having `<br/>` tags everywhere.